tests/handlers/test_space_summary.py

```diff
@@ -570,3 +583,55 @@ async def summarize_remote_room(
                 (subspace, joined_room),
             ],
         )
+
+    def test_fed_invited(self):
+        """
+        Rooms returned over federation should be properly filtered to only include
+        rooms the user has access to.
```

Comment:
this is the same as the docstring on `test_fed_filtering` - I suspect it has been c&p'ed and could do with an update?